modules/appeals_api/app/swagger/legacy_appeals/v0/api_description.md

```diff
@@ -1,18 +1,18 @@
-The Legacy Appeals API returns a list of a claimant's active legacy appeals, which are not part of the Appeals Modernization Act (AMA) process. This list can be used to determine whether to opt in to the new decision review process. [Learn more about managing a legacy appeal.](https://www.va.gov/decision-reviews/legacy-appeals/).
+The Legacy Appeals API returns a list of a claimant's active legacy appeals, which are not part of the Appeals Modernization Act (AMA) process. This list can be used to determine whether to opt in to the new decision review process. [Learn more about managing a legacy appeal](https://www.va.gov/decision-reviews/legacy-appeals/).
```

Comment:
This change moved the `.`, which I think should be part of the link (as it was before).

Reply:
I did this on purpose - I don't think periods are supposed to be included in links in cases where the link text doesn't include the entire sentence - here are two places I found that mention it:
* https://ux.stackexchange.com/questions/17331/should-a-full-sentence-html-link-include-the-period-in-the-linked-text
* https://www.chicagomanualofstyle.org/qanda/data/faq/topics/URLs/faq0007.html

That said, google's style guide suggests avoiding this confusion by moving the link out of the main text entirely: https://developers.google.com/style/periods#periods-with-urls - it seems like we'd need to go through UX to take that route, though